### ISSUE TITLE
About pretrain scheduling

### ISSUE THREAD

**Issue by sungyubkim** [Sha-Lab/FEAT#19]
Thanks for sharing the nice paper and implementations.

I have a question for your shared code pretrain.py

It seems that your default argparse setting of pretraining is set to 1e-3 and decay 0.1 in 30, 50, 80 epochs.

I found that it yields lower accuracy of pretrain phase train/valid/test in MiniImageNet or TieredImageNet.

Is there any special reason to set you 1e-3 for initial learning rate or bigger lr (e.g. 1e-1) will works better?

**Comment by Han-Jia** [comment]
For MiniImageNet, please use the learning rate 1e-1 as the initialization.  1e-3 is not used for this dataset, and I'll update this in the code later.
